samplesheets/views_api.py

```diff
@@ -310,6 +323,240 @@ def post(self, request, *args, **kwargs):
         return Response(ret_data, status=status.HTTP_200_OK)
 
 
+class IrodsAccessTicketListAPIView(
+    IrodsAccessTicketModifyMixin, SODARAPIBaseProjectMixin, APIView
+):
+    """
+    List iRODS access tickets for a project.
+
+    **URL:** ``/samplesheets/api/irods/ticket/list/{Project.sodar_uuid}``
+
+    **Methods:** ``GET``
+
+    **Query parameters:** ``active`` (boolean, default ``0``)
+
+    **Returns:**
+
+    - ``count``: Number of iRODS access tickets
+    - ``tickets``: List of iRODS access tickets
+    """
+
+    http_method_names = ['get']
+    permission_required = 'samplesheets.edit_sheet'
+
+    def get(self, request, *args, **kwargs):
```

Comment:
We should only have to override `get_queryset()` in this class, where we filter according to the `active` arg.

If returning a list instead of a `QuerySet` does not work, you'd have to replace the call for `is_active()` with `Q` operators for the active date querying. However, since we don't do any further filtering in the view, I guess simply copying the current code and returning a list should work..